include/ccf/http_responder.h

```diff
@@ -36,17 +36,14 @@ namespace ccf::http
     {
       nlohmann::json body = ccf::ODataErrorResponse{
         ccf::ODataError{std::move(error.code), std::move(error.msg)}};
-      const auto s = body.dump();
+      const std::string s = body.dump();
```

Comment:
Unfortunate that we can't serialise straight to a vector of uint8_t and have this copy...